README.md

```diff
@@ -1,116 +1,76 @@
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/netmiko.svg)
 [![PyPI](https://img.shields.io/pypi/v/netmiko.svg)](https://pypi.python.org/pypi/netmiko)
 [![Downloads](https://pepy.tech/badge/netmiko)](https://pepy.tech/project/netmiko)
+![GitHub contributors](https://img.shields.io/github/contributors/ktbyers/netmiko.svg)
 [![Code Style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
 
 
 Netmiko
 =======
 
-Multi-vendor library to simplify Paramiko SSH connections to network devices  
+Multi-vendor library to simplify Paramiko SSH connections to network devices
 
-Python 2.7, 3.5, 3.6, 3.7  
+## Quick Links
 
-#### Requires:
+- [Supported Platforms](SupportedPlatforms)
+- [Installation](Installation)
+- [Documentation](Documentation)
+- [Tutorials/Examples](#TutorialsExamples)
+- [Common Issues/FAQ](#Common-IssuesFAQ)
+- [TextFSM Integration](#TextFSM-Integration)
+- [Contributing](#Contributing)
+- [Questions/Discussion](#QuestionsDiscussion)
 
+
+## Supported Platforms
+
+Netmiko supports a wide range of devices. These devices fall into three categories:
+- Regularly Tested
+- Limited Testing
+- Experimental
+
+Devices in the "Limited Testing" and "Experimental" categories of course have much less testing and are not garuanteed to be 100% functional, so please take care when interacting with those devices.
```

Comment:
Spelling guaranteed. Also I would change that text as it implies a bit that the regularly tested are guaranteed in some way. Regularly tested means we try to run our full test suite against that set of devices prior to each Netmiko release. Limited testing means the `config` and `show` operation system tests passed against a test on that platform at one point in time so we are reasonably comfortable the driver should generally work. Experimental means that we reviewed the PR and the driver seems reasonable, but we don't have good data on whether the driver fully passes the unit tests or how reliably it works.

Reply:
updated!